build/packages/rpm/server/SPECS/pmm-update.spec

```
@@ -63,6 +63,9 @@ install -p -m 0755 bin/pmm-update %{buildroot}%{_sbindir}/
 
 
 %changelog
+* Thu Dec 8 2022 Michal Kralik <michal.kralik@percona.com> - 2.29.0-2
```

I think version here should be 2.34.0
```suggestion
* Thu Dec 8 2022 Michal Kralik <michal.kralik@percona.com> - 2.34.0-67
```